packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -110,6 +110,7 @@ export const TabsItem = compose<TabsItemType>({
         accessibilityPositionInSet: accessibilityPosInSet ?? info.tabsItemKeys.findIndex(x => x == itemKey) + 1,
         accessibilitySetSize: accessibilitySetSize ?? info.tabsItemKeys.length,
         onAccessibilityAction: onAccessibilityAction,
+        focusable: Platform.OS=='macos' ? !userProps.disabled : userProps.focusable,
```

Comment:
Does focus not get disabled when disabled is set?

Reply:
This makes it so that a disabled TabsItem cannot be focusable

Reply:
Did you not need to do this for win32 or windows? If so, that sounds like it could be a bug we open on react-native-macos

Reply:
here's the bug I filed for this: https://office.visualstudio.com/OC/_workitems/edit/5382366